app/facades/cocktail_facade.rb

```diff
@@ -1,6 +1,7 @@
 class CocktailFacade
   def self.retrieve_cocktail(user_id, cocktail_id)
-    return false if retrieve_details(cocktail_id) == {:drinks=>nil}
+    return false if retrieve_details(cocktail_id) == { drinks: nil }
```

Comment:
When I tried it this way, my tests were not passing so not sure what I was doing wrong...

Reply:
I think it is because I added/updated the retrieve_details method possibly?

Reply:
@markcyen yea, it's because now my retrieve_details method deals with nil values, so this can be more specific